pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -29,6 +29,7 @@ func InitReportAnalyticsWorkflow(engine workflow.Engine) error {
 	// initialise workflow configuration
 	params := pflag.NewFlagSet(reportAnalyticsWorkflowName, pflag.ExitOnError)
 	params.StringP(reportAnalyticsInputDataFlagName, "i", "", "Input data containing scan done event")
+	params.Bool(experimentalFlag, false, "enable experimental analytics report command")
```

Comment:
Suggestion: maybe it makes sense to move this constant to `configuration.Experimental`